### PR TITLE
[ros2] Fix RCLCPP_* logs, a string literal must be the first argument

### PR DESCRIPTION
**Pull request by ivanpauno**
The `ros2` branch doesn't compile with neither rolling or galactic.
This PR fixes it.

### CODE REVIEW

**Review by jacobperron — APPROVED** [review]

**Comment by ivanpauno** [comment]
@jacobperron I'm not sure if you want to merge this already or if we're waiting for another review.
FYI, I don't have write access in this repo.

**Comment by jacobperron** [comment]
Pinging @mjcarroll as maintainer.

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
